src/components/dspPricing/pricingPlanData.json

```diff
@@ -0,0 +1,214 @@
+{
```

Comment:
I used a `json` file to capture all the details listed within the Figma design. 

1. I didn't see us using json for any other occasion so please let me know if there is a better or more code base consistent way to implement this data set.
2. I will double check with the Backend team to see if any of these details might come to the Frontend via an endpoint. From my understanding it is only the option to save the users DSP choice, but pricing from an endpoint might make sense here as well.